vendor/modules.txt

```diff
@@ -157,14 +155,6 @@ github.com/nxadm/tail/winfile
 github.com/onsi/ginkgo
 github.com/onsi/ginkgo/config
 github.com/onsi/ginkgo/formatter
-github.com/onsi/ginkgo/ginkgo
```

Comment:
For clarification, these are used for tests.

Reply:
This is not nice, as one would think this dependency is not required.
I have updated PR, now `github.com/onsi/ginkgo/ginkgo` is considered as a dependency for this project. So next time if someone runs `go mod tidy && vendor` it will not drop `github.com/onsi/ginkgo/ginkgo` from vendor directory.

Reply:
I have kept the file that forces `github.com/onsi/ginkgo/ginkgo` as a dependency for this project 95056202c6e6c83b6d7aa9997034807c1c9ef2a3. If there is some better suggestion I will adapt the PR accordingly.